### PR TITLE
K8s 1 15 1

### PR DESCRIPTION
**Pull request by qinqon**
This bump kubevirtci provider used k8s to 1.15.1

### CODE REVIEW

**Comment by phoracek** [comment]
@qinqon please rebase

**Comment by qinqon** [comment]
ci test please

File: cmd/manager/main.go [hunk 1]
```diff
@@ -160,11 +160,11 @@ func serveCRMetrics(cfg *rest.Config) error {
 	if err != nil {
 		return err
 	}
-
-	// To bypass ugly metrics logs after operator-sdk 0.12.0 upgrade
-	// https://github.com/operator-framework/operator-sdk/issues/1858#issuecomment-548323725
-	operatorNs := ""
-
+	// Get the namespace the operator is currently deployed in.
```
Comment (phoracek) [inline cmd/manager/main.go]: How is this change related?

Comment (qinqon) [reply cmd/manager/main.go]: https://github.com/operator-framework/operator-sdk/issues/1858#issuecomment-525068185

Comment (phoracek) [reply cmd/manager/main.go]: We need to make sure it is not reproducible on OKD-4.3. I don't remember what Kubernetes version is used there.

Comment (qinqon) [reply cmd/manager/main.go]: Ok going to remove the revert, looks like it's still not working.

**Comment by qinqon** [comment]
ci test please
